Example/ViewController.m

```diff
@@ -16,38 +18,23 @@ @interface ViewController ()
 @property (nonatomic, strong) DMEPullClient *dmeClient;
 @property (nonatomic, strong) LogViewController *logVC;
 @property (nonatomic, strong) DMEPullConfiguration *configuration;
+@property (nonatomic, strong) DMEOAuthObject *accessToken;
```

Comment:
Please change property name to `oAuthToken` to more accurately represent it's type.